packages/appeals-service-api/src/services/document-metadata.service.js

```diff
@@ -19,6 +25,17 @@ const getDocumentMetadata = async (caseRef, documentType, returnMultipleDocument
 	return document;
 };
 
+const createOrUpdateDocumentMetadata = async (documentMetadata) => {
+	// check schema with joi
+	const validationResult = documentMetaDataSchema.validate(documentMetadata);
+	if (validationResult.error) {
+		throw ApiError.badRequest();
+	}
+
+	await documentMetadataRepository.createUpdateDocumentMetadata(documentMetadata);
```

Comment:
If you just return this function call then this function won't need any async/await sugar but will still work the same way